MiniProjectUIKit/ViewController/HomepageViewController.swift

```diff
@@ -0,0 +1,285 @@
+//
+//  HomepageViewController.swift
+//  MiniProjectUIKit
+//
+//  Created by Vincent Saranang on 06/12/24.
+//
+
+import UIKit
+
+class HomepageViewController: UIViewController, UISearchBarDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
+
+    let searchBar = UISearchBar()
+    let filterScrollView = UIScrollView()
+    var collectionView: UICollectionView!
+    let titleLabel = UILabel()
+    let activityIndicator = UIActivityIndicatorView(style: .large)
+    
+    var areas: [String] = []
+    var selectedFilters: Set<String> = []
+    var meals: [Meal] = []
```

Comment:
sepertinya better ini nama-nya, lebih menggambarkan

```suggestion
    var filteredMeals: [Meal] = []
```